stl/inc/xstring

```diff
@@ -4028,6 +4159,20 @@ public:
     }
 #endif // _HAS_CXX17
 
+#ifdef __cpp_lib_containers_ranges
+private:
+    constexpr void _Push_back_unchecked(const _Elem _Ch) { // insert element at end; assume available capacity

```

Comment:
It does not seem useful to create this function, as it is called exactly once. IMO, we should call this in other places (like `push_back`) where this exact code is used, or just take the code and paste it into the calling site.

Additionally, this function fails to call `_ASAN_STRING_MODIFY`.

Reply:
Yeah, this seems to be part of a larger refactoring that I gave up on for fear of merge conflicts with the constexpr SSO PR. I'm going to burninate this and the `from_range` constructor and rewrite them in conjunction with the iterator-pair constructor.